airbyte-integrations/connectors/source-typeform/source_typeform/source.py

```diff
@@ -80,7 +80,7 @@ def request_params(
         params["page"] = next_page_token or 1
         return params
 
-
+# same loop logic
 class TrimFormsMixin:
     def stream_slices(self, **kwargs) -> Iterable[Optional[Mapping[str, any]]]:
         form_ids = self.config.get("form_ids", [])
```

Comment:
If user pass empty form list will trigger full refresh that will list all forms again.

Reply:
we can remove this logic, or put required form id in frontend and change the logic.

Reply:
Here is necessary this logic.

Reply:
sorry, this is the lines that i don't agree with

https://github.com/airbytehq/airbyte/pull/8466/files#diff-b564154cf70574820f36c0953c3ae6943851423f132e7ed11ab8b5c2db32a14fR90-R92

Reply:
sorry, those are the lines i don't agree with
https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-typeform/source_typeform/source.py#L90-L92

Reply:
but i'ts necessary if someone want to get all forms and don't provide a list of forms